### ISSUE TITLE
AWS options are double nested in the elasticsearch sink

### ISSUE THREAD

**Issue by binarylogic** [timberio/vector#1099]
## Problem

See https://github.com/timberio/vector/issues/1093. If a user is using the `elasticsearch` sink with the `provider` set to `aws` the user must do the following in order to set the AWS `region`:

```toml
[sinks.elasticsearch]
  type = "elasticsearch"
  provider = "aws"

  # The region has to be provided like this, the documentation appears to be incorrect
  [sinks.elasticsearch.region]
    region = "eu-west-1"
```

Notice the double `region` nesting.

## Solution

I'd prefer that we fix this with #1084:

```toml
[sinks.elasticsearch]
  type = "elasticsearch"

  [sinks.elasticsearch.auth]
    strategy = "aws"
    region = "eu-west-1"
    # ... other AWS specific options here
```

**Comment by binarylogic** [comment]
@bruceg let me know if you have any questions about the above change.

**Comment by bruceg** [comment]
I have resolved the nested `region` setting in #1116. Does this resolve this issue or should it stay open for #1084?

**Comment by binarylogic** [comment]
Thanks! We can close this issue for now and address the `auth` namespace in #1084.
